chip_width: 640
chip_height: 480
img_pub_topic_name: /image_raw
img_pub_queue_size: 10
